### PR TITLE
Fix kubectl-skew jobs against master

### PR DESCRIPTION
**Pull request by pwittrock**
- separate serial / non-serial tests
- stop running cvm, only run gci

### CODE REVIEW

File: jobs/config.json [hunk 1]
```diff
@@ -2539,27 +2539,27 @@
...
     "args": [
       "--check-leaked-resources",
       "--check-version-skew=false",
       "--env-file=jobs/platform/gce.env",
-      "--env-file=jobs/ci-kubernetes-e2e-gce-master-new-cvm-kubectl-skew.env",
+      "--env-file=jobs/ci-kubernetes-e2e-gce-master-new-gci-kubectl-skew.env",
       "--extract=ci/latest-1.7",
       "--extract=ci/latest",
-      "--gcp-project=gce-cvm-upg-1-5-1-4-ctl-skew",
+      "--gcp-project=gce-gci-upg-1-5-1-4-ctl-skew",
```
Comment (krzyzacy) [inline jobs/config.json]: now gci-skew and gci-skew-serial are sharing the same projects - you can remove both `--gcp-project` flags and add the projects to https://github.com/kubernetes/test-infra/blob/master/boskos/resources.json

**Comment by pwittrock** [comment]
/test pull-test-infra-bazel
--

**Comment by krzyzacy** [comment]
ah, also rename/shuffle stuff from https://github.com/kubernetes/test-infra/blob/master/prow/config.yaml and https://github.com/kubernetes/test-infra/blob/master/testgrid/config/config.yaml

File: jobs/config.json (hunk 1 above)
Comment (pwittrock) [inline jobs/config.json]: I think it should work as is not.  Mind if I do that as a follow up to reduce the number of things I am changing at once?

**Comment by pwittrock** [comment]
Yes, I will update the test grid next.  Thanks for the reminder.

File: jobs/config.json (hunk 1 above)
Comment (krzyzacy) [inline jobs/config.json]: yeah keep them would be fine, I'll handle the migration later.

**Comment by pwittrock** [comment]
@krzyzacy Any idea how the expected jobs are populated?  I don't understand if it is not finding it and things it should, or the reverse.

**Comment by krzyzacy** [comment]
You need to add the new ones as prow jobs first, if that's what you are asking.

**Comment by pwittrock** [comment]
@krzyzacy 

I am asking about this error from [the logs](https://k8s-gubernator.appspot.com/build/kubernetes-jenkins/pr-logs/pull/test-infra/3831/pull-test-infra-bazel/5352/):

`AssertionError: 'ci-kubernetes-e2e-gce-master-new-gci-kubectl-skew-serial' not found in ['ci-cadvisor-canarypush'`

**Comment by pwittrock** [comment]
I guess I have to update prow/config.yaml?

**Comment by pwittrock** [comment]
Looks like things are broken at head as well

**Comment by pwittrock** [comment]
Or maybe just on Macs?

File: prow/config.yaml [hunk 2]
```diff
@@ -5211,11 +5212,11 @@ periodics:
         secretName: ssh-key-secret
 
 - interval: 1h
-  name: ci-kubernetes-e2e-gce-new-master-gci-kubectl-skew
+  name: ci-kubernetes-e2e-gce-new-master-gci-kubectl-skew-serial
   spec:
     containers:
     - args:
-      - --timeout=140
+      - --timeout=90
```
Comment (krzyzacy) [inline prow/config.yaml]: this was what the timeout is complaining :-) keep it 140, or reduce the one from config.json

File: jobs/config.json [hunk 3]
```diff
@@ -6818,39 +6818,39 @@
...
     ]
   },
-  "ci-kubernetes-e2e-gke-new-master-cvm-kubectl-skew": {
+  "ci-kubernetes-e2e-gke-new-master-gci-kubectl-skew": {
     "args": [
       "--check-leaked-resources",
       "--check-version-skew=false",
       "--env-file=jobs/platform/gke.env",
-      "--env-file=jobs/ci-kubernetes-e2e-gke-new-master-cvm-kubectl-skew.env",
+      "--env-file=jobs/ci-kubernetes-e2e-gke-new-master-gci-kubectl-skew.env",
```
Comment (krzyzacy) [inline jobs/config.json]: oh oh oh, you'll need to rename this env file as well :-)

Comment (krzyzacy) [inline jobs/config.json]: Once we merge https://github.com/kubernetes/test-infra/pull/3819 we can get rid of those env files entirely.

**Comment by pwittrock** [comment]
@krzyzacy PTAL

**Pull request merged**